docs/topics/installing-vs-code-plugin.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * docs/vsc-extension-guide/master.adoc
+
+[id="installing-vs-code-extension_{context}"]
+= Installing the {ProductShortName} extension for Visual Studio Code
+
+You can install the {ProductShortName} extension for Visual Studio Code (VS Code) on any platform that supports VS Code.
```

Comment:
I'm not sure we need "on any platform".... I think it's probably enough to say "You can install the x extension for y."